site/src/pages/[shortname]/blocks/[block-slug].page.tsx

```diff
@@ -165,9 +177,17 @@ export const getStaticProps: GetStaticProps<
 
   const { schema } = await readBlockDataFromDisk(blockMetadata);
 
+  const readmeMd = await fetchBlockAsset(blockMetadata, "README.md");
+
+  const compiledReadme = readmeMd
+    ? (await serialize(readmeMd, { mdxOptions: { format: "md" } }))
```

Comment:
`{ format: "md" }` is unblocked by #321. Related: https://github.com/hashicorp/next-mdx-remote/issues/279